tests/regression/38-int-refinements/02-strange-ulong.c

```diff
@@ -0,0 +1,86 @@
+// PARAM: --enable ana.int.interval --set ana.int.refinement once
+#include <assert.h>
+
+int main() {
+  withint();
+  withuint();
+  withlong();
+  withlonglong();
+  withulonglong();
```

Comment:
Using functions before they were defined or declared produces a warning, so one might move the `main` down.